.github/workflows/main.yml

```diff
@@ -63,12 +64,17 @@ jobs:
       with:
         name: conda-bld
         path: conda-bld
+    - name: Setup Miniconda
+      uses: conda-incubator/setup-miniconda@a4260408e20b96e80095f42ff7f1a15b27dd94ca
+      with:
+        miniconda-version: 'latest'
+        auto-activate-base: true
+        activate-environment: ""
+        conda-solver: classic

```

Comment:
classic since that would make it work with the older versions of conda that are being tested